Documents/cluster.md

```diff
@@ -3,4 +3,28 @@
 ## Cluster Management
 
 The project uses the [Akka Cluster HTTP Management](http://developer.lightbend.com/docs/akka-management/current/cluster-http-management.html) module to report on cluster member status as well as to perform cluster
-management tasks. Please refer to the documentation of this module for cluster management tasks (i.e. downing a node from the cluster)
\ No newline at end of file
+management tasks. Please refer to the documentation of this module for cluster management tasks (i.e. downing a node from the cluster)
+
+## Roles
+
+There are four different cluster roles, each managed within the cluster project.
+
+### API
+
+The API cluster role is responsible for the four different APIs: Admin, Public, Filing, and the TCP Panel Loader
+
+### Persistence
+
+The persistence cluster role contains actors responsible for tracking and persisting information about submissions, institutions, and LAR validation
+
+### Query
+
+This cluster role contains actors responsible for interacting with the backend database (currently Cassandra)
+
+### Publication
+
+The publication role is responsible for creating and publishing the A&D reports from submitted LAR data
+
+## Running cluster roles
+
+The configuration setting `"akka.cluster.roles"` in `cluster/src/main/resources/application.conf` defines which cluster roles will start when the HMDA Platform starts.  By default, all four roles are set to start at the same time, but this can be changed to any single role or combination of roles.  Currently, this can only be done from within the config file; there is no environment variable override.
```

Comment:
The last statement is actually not true. The `HMDA_CLUSTER_ROLES` environment variable allows overriding the specific role(s) for a particular node.

Reply:
I thought I was going crazy, I knew there was some environment variable!  I forgot that it was defined in `HmdaPlatform.scala` rather than the `application.conf` file.